content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -28,6 +28,15 @@ This permits customization of Istio to operator specific requirements.
 
 1. Perform any necessary [platform-specific setup](/docs/setup/kubernetes/prepare/platform-setup/).
 
+    {{< tip >}}
+    Run following command to verify the Kubernetes cluster is ready for Istio installation:
+
+    {{< text bash >}}
+    $ istioctl verify-install
```

Comment:
Can you add something to indicate what is the expected output?

Reply:
done